mediagrains/grain_constructors.py

```diff
@@ -36,8 +36,8 @@
 
 
 def Grain(src_id_or_meta=None, flow_id_or_data=None, origin_timestamp=None,
-          sync_timestamp=None, rate=Fraction(25, 1), duration=Fraction(1, 25),
-          flow_id=None, data=None, src_id=None, meta=None):
+          sync_timestamp=None, creation_timestamp=None, rate=Fraction(0, 1), duration=Fraction(0, 1),
+          flow_id=None, data=None, src_id=None, source_id=None, meta=None):
```

Comment:
I assume there is a `src_id_or_meta` and `src_id` to allow either a positional source id or a named parameter src_id. It's unclear to me why there is a also a `source_id` parameter.

Reply:
Because we need the src_id name for backwards compatibility, but source_id is the actual name of the attribute as it appears in the grain class.

Reply:
Worth adding a comment to explain this - since it is not immediately obvious from the function sig.